.github/actions/configure-wasi/action.yml

```diff
@@ -0,0 +1,37 @@
+name: ./configure
+inputs:
+  configurationParameters:
+    default: ''
+    required: false
+runs:
+  using: composite
+  steps:
+    - shell: bash
+      run: |
+        set -x
+
+        ./buildconf --force
+        ./configure \
```

Comment:
data extensions like xml, sqlite, openssl, ... must to be supported

opcache - it is not only for caching, but for runtime optimization too, it must compile
fiber-asm - fibers are core part of php and must be supported

Reply:
I'm going to update the PR to support xml and sqlite; as well as skipping tests individually as opposed to whole suites. I have to check openssl's feasibility.

WRT opcache, as well as fiber-asm; would it be possible to merge an initial version without them and iterate on that work, or are they a hard requirement for the very first version as well?